scripts/test.sh

```diff
@@ -559,22 +559,42 @@ function dep_pass {
 
 function release_pass {
   rm -f ./bin/etcd-last-release
+  mkdir -p ./bin
   # to grab latest patch release; bump this up for every minor release
-  UPGRADE_VER=$(git tag -l --sort=-version:refname "v3.5.*" | head -1 | cut -d- -f1)
-  if [ -n "${MANUAL_VER:-}" ]; then
+  UPGRADE_VER_LAST=$(git tag -l --sort=-version:refname "v3.5.*" | head -1 | cut -d- -f1)
+  if [ -n "${MANUAL_VER_LAST:-}" ]; then
     # in case, we need to test against different version
-    UPGRADE_VER=$MANUAL_VER
+    UPGRADE_VER_LAST=$MANUAL_VER_LAST
   fi
-  if [[ -z ${UPGRADE_VER} ]]; then
-    UPGRADE_VER="v3.5.0"
-    log_warning "fallback to" ${UPGRADE_VER}
+  if [[ -z ${UPGRADE_VER_LAST} ]]; then
+    UPGRADE_VER_LAST="v3.5.0"
+    log_warning "fallback to" ${UPGRADE_VER_LAST}
   fi
 
-  local file="etcd-$UPGRADE_VER-linux-$GOARCH.tar.gz"
+  download_etcd_ver_to_tem ${UPGRADE_VER_LAST}
+  mv /tmp/etcd ./bin/etcd-last-release
+
+  UPGRADE_VER_BEFORE_LAST=$(git tag -l --sort=-version:refname "v3.4.*" | head -1 | cut -d- -f1)
+  if [ -n "${MANUAL_VER_BEFORE_LAST:-}" ]; then
+    # in case, we need to test against different version
+    UPGRADE_VER_BEFORE_LAST=$MANUAL_VER_BEFORE_LAST
+  fi
+  if [[ -z ${UPGRADE_VER_BEFORE_LAST} ]]; then
+    UPGRADE_VER_BEFORE_LAST="v3.4.0"
+    log_warning "fallback to" ${UPGRADE_VER_BEFORE_LAST}
+  fi
+
+  download_etcd_ver_to_tem ${UPGRADE_VER_BEFORE_LAST}
+  mv /tmp/etcd ./bin/etcd-before-last-release
+}
```

Comment:
@ahrtr trying to rebase from main. New logic uses minor version to determine prior release. But current [test](https://github.com/etcd-io/etcd/blob/f3cc759afc679983a609e462c2808265d989e2dd/tests/e2e/utl_migrate_test.go#L80) (even without my changes) assumes major version. It works right now because version in `main` is `3.6.0-alpha.0`, so it will get 3.5.X.
Shall I change the logic of the script to use major version?